pipeline/pipeline.go

```diff
@@ -496,25 +531,29 @@ func (p *Pipeline) maintenance() {
 		rate := int(float64(deltaCommitted) * float64(time.Second) / float64(interval))
 		rateMb := float64(deltaSize) * float64(time.Second) / float64(interval) / 1024 / 1024
 
+		// avoid division by zero.
 		tc := totalCommitted
 		if totalCommitted == 0 {
-			tc = 1
```

Comment:
Why the line is removed?) It's empty condition here

Reply:
I was in middle of refactoring. Moved it back.